Documentation/Signoffs/Raymond Mule-Signoff-Wireless Communication Subsystem(Updated).md

```diff
@@ -0,0 +1,73 @@
+
+The RAK811 transceiver has a very high sensitivity of -148 dBm, enabling it to transmit data over extremely long distances. This high sensitivity allows the transceiver to receive signals that have become very weak after traveling a great distance.
+
+## Schematic- 
+
+![Newest Schematic](https://github.com/Michaelwwest98/DARPA-Drone-Triage-Sensing-System/assets/123600489/cc0cd754-5744-4b14-8777-ad1688809484)
+
+| RAK811 LoRa Breakout Board PINOUT |---|
+|-----------------------------------|---|
+| Pin Number | Description |
+| 1 | 3.3 V |
+| 2 | UART1_TXD |
+| 3 | UART1_RXD |
+| 4 | PB12 |
+| 5 | RST |
+| 6 | PA3 |
+| 7 | PB5 |
+| 8 | PA15 |
+| 9 | BOOT0 |
+| 10 | GND |
+| 11 | PA0 |
+| 12 | PA1 |
+| 13 | SWCLK |
+| 14 | SWDIO |
+| 15 | PB11 |
+| 16 | PB15 |
+| 17 | PA2 |
+| 18 | PA13 |
+| 19 | PA12 |
+| 20 | PB14 |
+
+## Analysis-
+1.  The RAK811 transceiver has a range of 15 kilometers, which fulfills the constraint of a 9 kilometer distance.
+
+2. The data that will be sent from the computing system to the user interface will be approximately 30 bits in size according to the designer of the computing subsystem. The RAK811 has an average data rate of 300 kbps meaning that the data from the computing subsystem will be able to reach the user interface subsystem within the two second minimum time frame specified in the constraints.
+
+3.  The weight of the RAK811 LPWAN breakout module the RAKDAP1 debug and flash tool that will be attached to the sensing system weighs approximately 24.4 grams or 0.0538 pounds which is a signaficantly less than the maximum 1.2 pounds the entire subsystem must weigh.
+
+4. The RAK811 Breakout Board operates in the 915 Mhz band and the maximum power output that the RAK811 Breakout Board is 100mW or 20 dBm, which falls below the safety standard of 30 dBm or 1 watt set by the FCC Part 15 Subpart C.
+
+5. The RAK811 Breakout board as previously stated, must operate in the 915 MHz band which is the frequency band for United States. This means the bandwidth that is required for the device is either 125 kHz or 500 kHz. On the RAK811 data sheet, it states that the bandwidth of the device is able to be configured. The minimum bandwidth of the transceiver is 125 kHz and the maximum is 500 kHz. This fulfills the constraint for the bandwidth of the system.
```

Comment:
Again, what is the frequency band for the US? That statement does not make sense. Further, why is there a bandwidth constraint?